Microsoft.Toolkit.Uwp.Input.GazeControls/UIElementExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.Input.GazeControls
+{
+    internal static class UIElementExtensions
+    {
+        public static T FindControl<T>(this UIElement parent, string controlName)
```

Comment:
We already have Visual Tree helpers in the Toolkit's Uwp.UI package, just include a reference to that package and use those directly.

Reply:
Thanks. I will update.

Reply:
Done. And I have deleted my UIElementExtensions.cs file.